### ISSUE TITLE
Make the game easier

### ISSUE THREAD

**Issue by Jontpan** [AxlLind/2D-platformer#97]
From our user tests we have found that the game is simply too hard, and that you expect to reach further with each jump. The shortness of the jump may be our USP, or it may be stupid. If changed, we should probably also change camera speed.

**Comment by Jontpan** [comment]
The difficulty of our game turns a lot of users off. We should figure out a way to lower the barrier of entry somewhat, while still keeping our core gameplay as it is.

**Comment by AxlLind** [comment]
Yes, I agree. The difficulty of the game is what makes it fun but when you're just learning the controls it should be easier. Here's an idea:

What if say for the first 200 the distance between platforms is like half? Or 2 thirds? That would make it a lot easier in the begining. Then new players can get a feel for the movement and when they do it gets harder.

**Comment by Jontpan** [comment]
Yeah, just fill the screen up with platforms at the start. Maybe having multiple platforms on the same y? Just more stuff. Seems good.

**Comment by AxlLind** [comment]
Yeah, more platforms and the same y might be better than altering the distance. When I tried halfing the distance it looked really ugly.

**Comment by Jontpan** [comment]
Echoing what we talked about about having the jumping be slower and higher at the start, and gradually becoming the intense gaming experience it is now.

**Comment by Jontpan** [comment]
The game should be easy enough now, but I'll hold this open until we get some testers to confirm this.
